container.go

```diff
@@ -91,6 +92,36 @@ func WithTTY(in FdReader, out, err io.Writer) ContainerOption {
 			ch := make(chan os.Signal, 1)
 			ch <- syscall.SIGWINCH // Initial resize.
 
+			go func() {
+				defer pw.Close()
+				fd := int(in.Fd())
+				for {
+					var fds syscall.FdSet
+					fdZero(&fds)
+					fdSet(fd, &fds)
+					err := syscall.Select(fd+1, &fds, nil, nil, &syscall.Timeval{Usec: 100000})
+					if errors.Is(err, syscall.EINTR) {
+						continue
+					}
+					if err != nil {
+						return
+					}
+					select {
+					case <-ctx.Done():
+						return
+					default:
+					}
+					if fdIsSet(fd, &fds) {
+						var b [1024]byte
```

Comment:
1024 seems small.  4k or 16k might be good if we expect stdin to be used as a input pipe to a remote container.  Also should we allocate/reuse it outside the loop?